src/overlays/actors/ovl_En_Kanban/z_en_kanban.c

```diff
@@ -5,6 +5,7 @@
  */
 
 #include "z_en_kanban.h"
+#include "global.h"
```

Comment:
Why did you add global.h here and in other files?

Reply:
in main, z_fishing indirectly includes global.h through including  z_en_kanban.h

As a first step towards not having to deal with z_fishing bss every single PR ever, I explicitly include global.h in z_fishing.h (goal: remove it eventually) and remove it from z_en_kanban.h (but have to add it back to z_en_kanban.c)